src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="card" data-qa="card">
+        <div class="card__logo-container">
+          <img
+            class="card__logo card__logo--position_center"
+            src="images/imac.jpeg"
+            alt="logo"
```

Comment:
В `alt` можна вписати "Моноблок APPLE....", це буде краще для скрінрідерів та якщо картинка не зможе підгрузитись буде більш зрозуміло, що саме там мало бути

```suggestion
            alt="Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)"
```